src/app/lib/presentation/diagnostic/selectable_diagnostic_button.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:app/presentation/diagnostic/diagnostic_button.dart';
+import 'package:flutter/material.dart';
+
+final class SelectableDiagnosticButton extends StatelessWidget {
+  const SelectableDiagnosticButton({
+    super.key,
+    required this.label,
+    required this.onPressed,
+    required this.isSelected,
+  });
+
+  final String label;
```

Comment:
Wondering if they should be private, same question for other widget, but I guess it doesn't matter too much right?

Reply:
I mean, the only thing this does, is it would allow someone to see the value of this property is from the outside, since it's final anyway. Like, I imagine the color value of a textbox in .net is public, no? I think it's fine like this.

Reply:
Yeah I think it's fine but I also thing that when we create a class and create fields / properties that won't be accessed from the outside we usually use private

Reply:
Your choice